docs/Developers/Newcomers_Guide.rst

```diff
@@ -152,7 +186,25 @@ Now you can start working on it.
 
         cobot assign <issue_link>
 
-    Take care to write the full link to the issue
+    Take care to write the full link to the issue.
+    Also take up the issue, only when you know what the problem is and
+    how to solve it.
+
+    You can do amazing stuff using cobot.
+
```

Comment:
I think we should add a short note here about testing cobot at the cobot-test repo.
Thoughts?